api/src/main/java/org/openmrs/Program.java

```diff
@@ -122,14 +126,63 @@ public String toString() {
 	// Property Access
 	// ******************
 	
+	/**
+	 * @deprecated since 2.1.0 replaced with {@link getName() and getDescription()}
```

Comment:
@dkayiwa doesn't deprecating mean annotating the code but keeping it for backwards compatibility or waiting for deletion during a new release?
a little puzzled here